<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>빔포밍 테스트</string>
  </property>
  <widget class="QPushButton" name="btn_Tx_en">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Tx Enable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Rx_en">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Rx Enable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Tx_dis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Tx Disable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Rx_dis">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Rx Disable</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SFR_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="BOARD_comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_SFR_write">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>SFR Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SFR_read">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>SFR Read</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SFR_address">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ARTIX_select">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Select ARTIX</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="SFR_output">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Stop">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>640</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rx_cal_comp_bp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Rx reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Rx_cal_board">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Rx calibration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Tx_cal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Tx calibration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Tx_reset">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Tx reset</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ARTIX_comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_BOARD_select">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Select BOARD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Retro">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Retro Continuous</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="value_output1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>361</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="value_output2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>361</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Retro_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>570</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Retro Single</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
